README.md

```diff
@@ -24,10 +28,9 @@ azd up
     npx @modelcontextprotocol/inspector
     ```
 
-2. CTRL click to load the MCP Inspector web app from the URL displayed by the app (e.g. http://127.0.0.1:6274/#resources)
-1. Make sure update the host to **localhost** (e.g. http://localhost:6274/#resources)
```

Comment:
I don't need to change the host to `localhost` when testing. So removed this step. Feel free to add it back if it's needed for some scenarios.